src/main/java/com/google/sps/data/CourierStop.java

```diff
@@ -0,0 +1,48 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package com.google.sps.data;
+
+import com.google.maps.model.LatLng;
+import com.google.maps.errors.ApiException;
+import com.google.sps.data.MapsRequest;
+import java.io.IOException;
+import java.lang.InterruptedException;
+import java.util.List;
+import java.util.ArrayList;
+
+/**
+ * Class that represents a stop in a delivery journey.
+ */
+public class CourierStop {
+  Point point;
+  boolean isLibrary;
```

Comment:
Ah sorry you don't need this boolean. I forgot we have "LibraryPoint extends Point".

At some point we might want to get rid of that, but until then, this is redundant and unused information. You can already infer that via (point instanceof LibraryPoint) if you ever need that info

Reply:
I removed it.